src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/NotificationsService.cs

```diff
@@ -67,6 +67,10 @@ await _context.Entry(release)
                 latestUpdateNoteReason = latestUpdateNote.Reason;
             }
 
+            var supersededPublications = _context.Publications
+                .Where(p => p.SupersededById == release.PublicationId)
+                .Select(p => new { p.Id, p.Title })
+                .ToList();
```

Comment:
I don't think so. A publication can only have one `SupersededById` set. There is a chance that it could fetch multiple different publications - i.e. a publication supersedes multiple publications - but that case is handled.

Reply:
Ah yep, I see now, they would all be different publications if there's more than one result. 👍